From 35c36a63321fcc5ce1de64520e9be9075ae510bd Mon Sep 17 00:00:00 2001
From: "Chen, Yu" <yu.y.chen@intel.com>
Date: Thu, 20 Sep 2018 15:22:12 +0800
Subject: [PATCH] [REVERTME] Disable kati check for below case

   "implicit_rules",
   "implicit_rules",
   "real_to_phony",
   "phony_looks_real",
   "writable"

Change-Id: I565459402d903bb3d61683f63f863f80641ac299
Signed-off-by: Chen, Yu <yu.y.chen@intel.com>
---
 ui/build/kati.go | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/ui/build/kati.go b/ui/build/kati.go
index 959d0bdf..0e971031 100644
--- a/ui/build/kati.go
+++ b/ui/build/kati.go
@@ -131,18 +131,18 @@ func runKatiBuild(ctx Context, config Config) {
 
 	// PDK builds still uses a few implicit rules
 	if !config.IsPdkBuild() {
-		args = append(args, "--werror_implicit_rules")
+		//args = append(args, "--werror_implicit_rules")
 	}
 
 	if !config.BuildBrokenDupRules() {
-		args = append(args, "--werror_overriding_commands")
+		//args = append(args, "--werror_overriding_commands")
 	}
 
 	if !config.BuildBrokenPhonyTargets() {
-		args = append(args,
-			"--werror_real_to_phony",
-			"--werror_phony_looks_real",
-			"--werror_writable")
+		//args = append(args,
+		//	"--werror_real_to_phony",
+		//	"--werror_phony_looks_real",
+		//	"--werror_writable")
 	}
 
 	args = append(args, config.KatiArgs()...)
-- 
2.21.0

